recipes/pyiron_database/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pyiron_database" %}
+{% set version = "0.0.8" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7577eada2ba9b8b7846f0336386060e1f8c5c99f6e632101608766ac8ea8b3ad
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - pip
+    - h5py
+    - versioneer
+    - toml
+    - pyiron_workflow
+  run:
+    - python >={{ python_min }},<3.13
+    - h5py ==3.12.1
+    - versioneer ==0.29
+    - pyiron_workflow ==0.12.0
```

Comment:
Do these need to be this precise?

Reply:
Right now, the pip package is with such tight bounds - at least for this version. It is probably possible to go down with some versions and it will be for sure possible to extend the upper range as well for newer versions of the package. We also have the interest to broaden the scope to allow for a most up to date version of several packages. Due to the pinned pip dependencies, however, I would leave it as is for this version.

Reply:
While that is bad practice, it is the correct metadata upstream. Merging this...

Reply:
Yep, I agree. I am sure the policy will align to our other packages (fixing a lower bound we know works and allowing to the most recent version of dependencies tested via dependabot). Thanks for the 2nd review and the merge!